lightning/src/ln/channelmonitor.rs

```diff
@@ -1275,6 +1300,14 @@ impl<ChanSigner: ChannelKeys> ChannelMonitor<ChanSigner> {
 		self.payment_preimages.insert(payment_hash.clone(), payment_preimage.clone());
 	}
 
+	pub(super) fn broadcast_latest_local_commitment_txn<B: Deref>(&mut self, broadcaster: &B)
```

Comment:
c6470ad

Maybe add a TODO, this should be plugged to OnchainTxHandler to register broadcast request. But I don't propose to do it in #540 because a) some murky refactor of OnchainTxHandler to extract registration out of block_connected b) going to remove a lot of tx duplicata in tests c) useless while we don't have a CPFP API